Client.xcodeproj/project.pbxproj

```diff
@@ -12351,14 +12351,14 @@
 			};
 			name = Release;
 		};
-		3B43E3D91D95C48E00BBA9DB /* Beta */ = {
+		3B43E3D91D95C48E00BBA9DB /* Development_TestFlight */ = {
```

Comment:
It's kind of counter-intuitive for me to find both words `Development` and `TestFlight` in the the name of the configuration. My brain associates the term `Development` with a `Debug` configuration, and when building for `TestFlight` usually a `Release`-like config is used. 

So for clarity, the term `Development` must be included, so that SPM in iOS-core does infer it as non-`#Production` environment? Did I catch that correctly?

Reply:
Thanks for the comment @ecotopian . I agree it does sound counter intuitive. I've gone through the reason of choosing this approach more thoroughly in the Core PR.
I was planning to update the README as well as I do think that indeed providing additional knowledge to it would concretely help clarifying.